.circleci/config.yml

```diff
@@ -1083,6 +1083,13 @@ jobs:
             JEST_JUNIT_OUTPUT_DIR: ./reports/
       - store_test_results:
           path: libs/ui/reports/
+      - when:
+          condition:
+            equal: [ main, << pipeline.git.branch >> ]
+          steps:
+            - run:
+                name: Export App Strings Catalog
+                command: echo 0 # TODO(kofi): export to versioned folder accessible from VxDesign
```

Comment:
Feel free to hold off on this for just a bit! I'm hashing out a few ideas in the spec that I'm writing, and I'm thinking of publishing to S3 and owning that as part of the VxDesign / multi-language data pipeline work.

Might also be worth a quick sync since we're reaching the touch points between our threads of work

Reply:
Perfect! I figured this would be a good data "hand-off" point for your export/import pipeline, so will leave this open for you to update once you've settled on the plan 🤝

Reply:
Awesome! And we can sync as needed once I've shared the spec out and you've had a chance to read it!